src/Screens/Profile/index.tsx

```diff
@@ -113,11 +113,11 @@ const Profile: FunctionComponent<ProfileProps> = ({ navigation }) => {
               {!!diveLogs.length ? t('DIVE_LOG_PHOTOS') : t('NO_DIVE_LOGS')}
             </Text>
             <View style={styles.photosContainer}>
-              {diveLogs.map((diveLog, index) => {
```

Comment:
should never use the array index as the key. defeats the purpose of using the key field which should be an attribute of the object

Reply:
Yes, but I think this is only ironclad if the list is going to be modified and an index can point to a wrong object. Anyway, it's an easy fix. We can just point to the divelog id.

Reply:
yeah, but the list is going to be modified eventually, as the user adds new dive logs for example. in general, it's just good to get in the habit of never using the index

Reply:
Not really, as we fetch the entire thing from the backend after creating a review anyway, but I get your point.

Reply:
it's actually about how the rendering happens on the client side. even if you fetch the new data, because none of the keys will have changed, react may decide that nothing has changed and so it shouldn't rerender those specific cells.

Reply:
Well, I'm not sure that's entirely correct because if the number of items to be rendered change, that will register as a change and react will re-render..